contributors/README.md

```diff
@@ -167,6 +168,26 @@ TBD
 
 TBD
 
+### Benchmark
+
+#### TiDB
+
+It's appreciate to write benchmark code and add benchmark data if the change affects performance. The [benchstat](https://godoc.org/golang.org/x/perf/cmd/benchstat) tool is conventionally used to format benchmark data for change descriptions.
+
+We could do is run the benchmarks over multiple runs ie
+
+```bash
+go test -run=^$ -bench=BenchmarkXXX -count=10
```

Comment:
which directory should we enter in order to run this command successfully?

Reply:
I think it's a common knowledge for go user.